datamodel/changelogs/0001/00_db_extensions.sql

```diff
@@ -1,2 +1,3 @@
 CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE EXTENSION IF NOT EXISTS hstore;
```

Comment:
The audit is not currently bundled. I would recommend to revisit this before adding this extension. Jon(b) is generally preferred over hstore nowadays.

Reply:
dropping hstore requires changes in the history viewer plugin

Reply:
Yes, but I am just saying that currently there is nothing in this repo related to the history viewer. It's up to the manager to add this extension, it should not be integrated by default if not use by default.

Reply:
https://github.com/teksi/wastewater/pull/249 I changed the extensions here and added the history viewer needs in a separate PR.

We can always switch to jsonb when we implement it in history viewer.

This is a 1-1 functionnality with QGEP.

Reply:
See https://github.com/teksi/wastewater/pull/249 for later comments.